api/python/provisioner/commands/deploy_vm.py

```diff
@@ -102,50 +102,6 @@ class DeployVM(Deploy):
     _run_args_type = run_args_type
     setup_ctx: Optional[SetupCtx] = None
 
-    def set_pillar_data(self, setup_type, targets):
```

Comment:
Why not delete the file?

Reply:
will delete it once it passes multiple rounds of testing.